plugin/db/driver.go

```diff
@@ -244,8 +244,8 @@ type MigrationInfo struct {
 }
 
 // placeholderRegexp is the regexp for placeholder.
-// Refer to https://stackoverflow.com/a/6222235/19075342, but we support "." and "*" for now.
-const placeholderRegexp = `[^\\/?%:|"<>]+`
+// Refer to https://stackoverflow.com/a/6222235/19075342, but we support "." for now.
+const placeholderRegexp = `[^\\/?%*:|"<>]+`
```

Comment:
Context for why `*` was allowed: https://github.com/bytebase/bytebase/pull/2861#discussion_r984419297